puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/deploy_katello_site.sh

```diff
@@ -1,16 +1,5 @@
 #!/bin/bash -xe
 
-# Build the site on the slave 
-ruby=2.0.0
-# RVM Ruby environment
-. /etc/profile.d/rvm.sh
```

Comment:
Will moving this out of the shell script prevent the rest of the test from working?  Each build step is going to spawn a new shell, so the environment won't be inherited from one to the next.